plan/point_get_plan.go

```diff
@@ -188,15 +188,19 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt) *PointGetP
 			return nil
 		}
 		p := newPointGetPlan(ctx, schema, tbl)
-		p.Handle = handleDatum.GetInt64()
+		var err error
+		p.Handle, err = handleDatum.ToInt64(ctx.GetSessionVars().StmtCtx)
```

Comment:
We do not need to convert int64 to int64.

Reply:
this check is handled in the `ToInt64()` function